composer.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "gathercontent/wp-importer",
-  "description": "Imports items from GatherContent to your wordpress site",
+  "name": "content-workflow/wp-importer",
+  "description": "Imports items from Content Workflow to your wordpress site",
   "version": "3.2.19",
   "type": "wordpress-plugin",
   "keywords": [],
-  "homepage": "http://www.gathercontent.com",
+  "homepage": "https://www.bynder.com/en/products/content-workflow/",
```

Comment:
❓ Incredibly minor, but is it worth excluding the `/en/` so it redirects to the users language?

I'm not even sure if they do redirect, and I'm scared to test it because I have to set the language system wide, so I'm scared I wouldn't be able to get it back to English again 😆